website/templates/include/comment_pane_template.mako

```diff
@@ -24,14 +24,15 @@
                 <form class="form">
                     <div class="form-group">
                         <span>
-                            <div class="form-control atwho-input" placeholder="Add a comment" data-bind="editableHTML: {observable: replyContent, onUpdate: handleEditableUpdate}, attr: {maxlength: $root.MAXLENGTH}" contenteditable="true"></div>
```

Comment:
At mentions won't work if the comment pane uses a `textarea` instead of a contenteditable `div`.